RuleDocumentation/UseCompatibleCommands.md

```diff
@@ -41,10 +41,14 @@ Platforms bundled by default are:
 | 5.1                | Windows Server 2016   | `win-8_x64_10.0.14393.0_5.1.14393.2791_x64_4.0.30319.42000_framework` |
 | 5.1                | Windows Server 2019   | `win-8_x64_10.0.17763.0_5.1.17763.316_x64_4.0.30319.42000_framework`  |
 | 5.1                | Windows 10            | `win-48_x64_10.0.17763.0_5.1.17763.316_x64_4.0.30319.42000_framework` |
-| 6.1                | Windows Server 2016   | `win-8_x64_10.0.14393.0_6.1.3_x64_4.0.30319.42000_core`               |
-| 6.1                | Windows Server 2019   | `win-8_x64_10.0.17763.0_6.1.3_x64_4.0.30319.42000_core`               |
-| 6.1                | Windows 10            | `win-48_x64_10.0.17763.0_6.1.3_x64_4.0.30319.42000_core`              |
-| 6.1                | Ubuntu 18.04 LTS      | `ubuntu_x64_18.04_6.1.3_x64_4.0.30319.42000_core`                     |
+| 6.2                | Windows Server 2016   | `win-8_x64_10.0.14393.0_6.2.4_x64_4.0.30319.42000_core`               |
+| 6.2                | Windows Server 2019   | `win-8_x64_10.0.17763.0_6.2.4_x64_4.0.30319.42000_core`               |
+| 6.2                | Windows 10            | `win-4_x64_10.0.17763.0_6.2.4_x64_4.0.30319.42000_core`               |
+| 6.2                | Ubuntu 18.04 LTS      | `ubuntu_x64_18.04_6.2.4_x64_4.0.30319.42000_core`                     |
+| 7.0                | Windows Server 2016   | `win-8_x64_10.0.14393.0_7.0.0_x64_3.1.2_core`                         |
+| 7.0                | Windows Server 2019   | `win-8_x64_10.0.17763.0_7.0.0_x64_3.1.2_core`                         |
+| 7.0                | Windows 10            | `win-4_x64_10.0.17763.0_6.2.4_x64_3.1.2_core`                         |
```

Comment:
Is it possible to state here, in line 46 and `UseCompatibleTypes.md`, which minor version of Windows 10 was used (in the format of e.g. 1903 or 1909)?